canton_dep.bzl

```diff
@@ -4,5 +4,5 @@
 canton = {
     "sha": "a419de7d2e50578edf4c15bd4ed8f57d49396fa9832a02e9b78ff7a409d54326",
     "url": "https://www.canton.io/releases/canton-open-source-2.7.0-snapshot.20230705.10732.0.v8e5bbd5f.tar.gz",
-    "local": False,
+    "local": True,
```

Comment:
why is this needed?

Reply:
@remyhaemmerle-da Do you know if this is needed as well as the arbitrary_canton_sha file?

Reply:
Yes, see README.

Reply:
I may have misunderstood, so let me clarify. Setting `local` to `True` is necessary for CI to actually use the "local" Canton that gets built when `arbitrary_canton_sha` exists (and is a valid Canton commit sha). If you have the file but local is false, it gets built by CI but is then ignored by Bazel.

I do not have any opinion on whether or not you need an arbitrary commit here or you could use a published snapshot.

Reply:
I cannot use a published snapshot until tomorrow, as I merged the canton changes today. However if I wait until tomorrow, the canton bump here will fail. Instead I'm opting to use a custom hash until the bump tomorrow, which I believe will delete the sha file and revert local to false. Am I correct in that assumption?

Reply:
You'll be the first to test it AFAIK, so who knows 🤷, but, yes, your assumption does described the intended and expected behaviour.